heudiconv/tests/test_utils.py

```diff
@@ -58,4 +66,27 @@ def test_json_dumps_pretty():
           'Mar  3 2017 10:46:13 by eja'
     # just the date which reveals the issue
     # tstr = 'Mar  3 2017 10:46:13 by eja'
-    assert pretty({'WipMemBlock': tstr}) == '{\n  "WipMemBlock": "%s"\n}' % tstr
\ No newline at end of file
+    assert pretty({'WipMemBlock': tstr}) == '{\n  "WipMemBlock": "%s"\n}' % tstr
+
+
+def test_load_json(tmp_path, capsys):
+    # test invalid json
+    icontent = u"I'm Jason Bourne"
+    ifname = "invalid.json"
+    invalid_json_file = tmp_path / ifname
+    invalid_json_file.write_text(icontent)
+
+    with pytest.raises(JSONDecodeError):
+        load_json(str(invalid_json_file))
+        captured = capsys.readouterr()
+        assert ifname in captured.out
+
+    # test valid json
+    vcontent = {"secret": "spy"}
+    vfname = "valid.json"
+    valid_json_file = tmp_path / vfname
+
+    with open(str(valid_json_file), "w") as vj:
+        json.dump(vcontent, vj)
```

Comment:
`create_tree` here as well

Reply:
would `create_tree` also be appropriate if the contents are a dictionary, not a string? (I used `save_json` instead)